### PR TITLE
Removed Python 2.6,3.3 & Django <1.8,1.9

### PR DESCRIPTION
**Pull request by adamn**
Python 2.6 and 3.3 are no longer supported.  Django <1.8 and 1.9 are no longer supported.

### CODE REVIEW

**Comment by dfunckt** [comment]
Hey @adamn, thanks for the PR! I see Travis complains about a missing import with pypy3/Django 1.10 -- any ideas why?

**Comment by dfunckt** [comment]
s/missing import/import error

**Comment by adamn** [comment]
https://code.djangoproject.com/ticket/25849 is a wontfix so Django won't work on PyPy3.  I've just removed it entirely since I can't imagine anybody will be using pypy3 with Django 1.8 and the next version of django-rules.  It can be re-introduced whenever pypy3 supports Python 3.4+.

**Comment by dfunckt** [comment]
I see, thanks for tracking this down.

There are a few places rules branches depending on Django or Python version (eg. runtests.py, contrib/admin.py, README, etc.) -- are you willing to hunt them down and include their cleanup in this PR? They're already annotated so it should only be a matter of removing the back-compat branches.

Otherwise looks good.

**Comment by adamn** [comment]
Yeah I think I got all of them.  Cheers.

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ docs/_build
 pip-log.txt
 .tox
 .eggs/
+.idea/
+build/
```
Comment (michael-k) [inline .gitignore]: empty line is missing

**Comment by vstoykov** [comment]
I think that pypy3 can be restored. The issue was because TravisCI was using old version of PyPy3. Now it should be resolved.
https://github.com/travis-ci/travis-ci/issues/6277

**Comment by dfunckt** [comment]
This is superseded by #82. Thank you!

**Pull request closed without merging**